jdbc/spring-cloud-starter-stream-source-jdbc/src/main/resources/META-INF/spring-configuration-metadata-whitelist.properties

```diff
@@ -1,3 +1,4 @@
 configuration-properties.classes=org.springframework.cloud.stream.app.jdbc.source.JdbcSourceProperties, \
-  org.springframework.cloud.stream.app.trigger.TriggerPropertiesMaxMessagesDefaultOne
+  org.springframework.cloud.stream.app.trigger.TriggerPropertiesMaxMessagesDefaultOne, \
+  org.springframework.boot.autoconfigure.jdbc.DataSourceProperties
```

Comment:
Not sure what you mean, but `DataSource` is the most important part for this kind of JDBC application.
Since we don't do anything with that manually we fully rely on the Boot's `DataSourceAutoConfiguration` and that is good one.
But without whitelisting that we make end-user to do some hard work to figure out how he can set URL and credentials for his DB.
Am I missing anything ?